xml/installation-installation-configure-deployer-repos.xml

```diff
@@ -0,0 +1,487 @@
+   the product repositories will automatically be made available to the client
+   nodes and these steps can be skipped. Similarly, if you have configured the
+   &lcm; node as an SMT mirror, the product repositories will automatically be
+   made available to the client nodes and these steps can be skipped.
+   Otherwise, it is sufficient to either copy the data (from a remote host or
+   the installation media), to mount the product repository from a remote
+   server via <literal>NFS</literal>, or to loop mount a copy of the
+   installation images.
+   </para>
+   <para>
+    The following product media needs to reside in the specified
+    directories:
+   </para>
+   <table>
+    <title>Local Product Repositories for &cloud;</title>
+    <tgroup cols="2">
+     <colspec colnum="1" colname="1" colwidth="40*"/>
+     <colspec colnum="2" colname="2" colwidth="60*"/>
+     <thead>
+      <row>
+       <entry>
+        <para>
+         Repository
+        </para>
+       </entry>
+       <entry>
+        <para>
+         Directory
+        </para>
+       </entry>
+      </row>
+     </thead>
+     <tbody>
+      <row>
+       <entry>
+        <para>
+         &productname; &productnumber; DVD #1
+        </para>
+       </entry>
+       <entry>
+        <para>
+         <filename>&www_dir;/repos/Cloud</filename>
+        </para>
+       </entry>
+      </row>
+     </tbody>
+    </tgroup>
+   </table>
+   <para>
+    The data can be copied by a variety of methods:
+   </para>
+   <variablelist>
+    <!-- sec.depl.adm_conf.repos.product.media -->
+    <varlistentry>
+     <term>Copying from the Installation Media</term>
+     <listitem>
+     <para>
+      We recommended using <command>rsync</command> for copying. If the
+      installation data is located on a removable device, make sure to mount
+      it first (for example, after inserting the DVD1 in the &admserv; and
+      waiting for the device to become ready):
+     </para>
+     <bridgehead renderas="sect4">&cloudos; DVD#1</bridgehead>
+     <screen>mkdir -p &www_dir;/repos/&sle_repo;-Pool
+mount /dev/dvd /mnt
+rsync -avP /mnt/ &www_dir;/repos/&sle_repo;-Pool/
+umount /mnt</screen>
+     <bridgehead renderas="sect4">
+      &productname; &productnumber; DVD#1
+     </bridgehead>
+     <screen>mkdir -p &www_dir;/repos/Cloud
+mount /dev/dvd /mnt
+rsync -avP /mnt/ &www_dir;/repos/Cloud/
+umount /mnt</screen>
+     </listitem>
+    </varlistentry>
+    <varlistentry>
+     <!-- sec.depl.adm_conf.repos.product.remote -->
+     <term>Copying from a Remote Host</term>
+     <listitem>
+      <para>
+       If the data is provided by a remote machine, log in to that machine and
+       push the data to the &admserv; (which has the IP address <systemitem
+       class="etheraddress">192.168.245.10</systemitem> in the following
+       example):
+      </para>
+      <bridgehead renderas="sect4">&cloudos; DVD#1</bridgehead>
+      <screen>mkdir -p &www_dir;/repos/&sle_repo;-Pool
```

Comment:
same here, we are mixing -Pool and media?

Reply:
Fixed